.github/workflows/rust.yml

```diff
@@ -0,0 +1,46 @@
+name: Rust
+
+on:
+  push:
+    branches: [ imlk-dev ]
```

Comment:
Can these be updated to master?

Reply:
This is a mistake and has been fixed now